qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -3254,6 +3255,83 @@ def test_concurrent_model_instance_load_sanity(self):
                     )
                     self.assertTrue(triton_client.is_server_ready())
 
+    def test_ensemble_config_overwite(self):
```

Comment:
I don't see the test case is doing anything with ensemble, should probably change the name of the test case.

Reply:
:+1: good call, updated.